addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -613,6 +613,8 @@ a.no-decoration {
     }
 
     .carousel-control-prev, .carousel-control-next {
+        height: 70%;
```

Comment:
Can you try to use more generic position instead of 70 15 32 28 % ?

Reply:
And have you tried the design as discussed yesterday:

e.g.
![image](https://user-images.githubusercontent.com/4848797/55933683-802a0800-5c2e-11e9-894e-30af2a546d82.png)